web-components/src/components/inputs/SelectTextFieldBase.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { SelectProps, Theme, useTheme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+import DropdownIcon from '../icons/DropdownIcon';
+import { DefaultStyleProps } from './FieldFormControl';
+import TextFieldBase from './TextFieldBase';
+
```

Comment:
I'm not proud of this, but I needed a Select input with our styles without formik context. We have an issue to split these out properly (noted in TextField.tsx), but I didn't want to spend a ton of time on it right now.

Reply:
Maybe put a TODO in the code?